modules/scalaz/src/test/scala/pureconfig/module/scalaz/equality/package.scala

```diff
@@ -0,0 +1,22 @@
+package pureconfig.module.scalaz
+
+import org.scalactic.Equality
+
+import scalaz.{ DList, Heap }
+
+package object equality {
+
+  implicit def dListEquality[A]: Equality[DList[A]] = new Equality[DList[A]] {
```

Comment:
Can you add a quick comment here and in the instance below explaining why you need to override the default equality?